api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -209,4 +217,11 @@ message ExtProcOverrides {
 
   // Set a different gRPC service for this route than the default.
   config.core.v3.GrpcService grpc_service = 5;
+  
+  // [#not-implemented-hide:]
+  // The Metadata field can be used to provide additional information
```

Comment:
nit: It is unclear whether this meant to replace the metadata or to override/update the map. I'm assuming the former. Please add a clarifying comment.

Reply:
what do you mean replace? This field has same semantics as per route metadata. The metadata struct is merged with dynamic metadata using standard proto merge semantics, IIRC

Reply:
clarified